tests/specs/scss/_contains.test.scss

```diff
@@ -0,0 +1,24 @@
+@include test-module("contains") {
+  // Testing char-at functions
+  @include test("contains [function] should return true if key exist.") {
+    $expect: true;
+    $testlist: body 5, heading (blue red, green yellow);
+    $weights: normal, bold, light, bolder, lighter, 100, 200, 300, 400, 500, 600, 700, 800, 900, inherit, inital;
+    $multiple-options-correct: 200, 500;
+
+    @include assert-equal(contains($testlist, green), $expect, "should return true.");
```

Comment:
Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.